lib/model/country_table.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:plante/l10n/strings.dart';
+import 'package:plante/logging/log.dart';
+import 'package:plante/model/country.dart';
+import 'package:plante/model/country_code.dart';
+
+class CountryTable {
+  static Country? getCountry(String? iso2Code) {
+    Country? result;
+    if (iso2Code != null) {
+      result = countryTable[iso2Code];
+      if (result == null) {
+        Log.w('CountryTable.dart: No country found for code $iso2Code');
+      }
+    }
+    return result;
+  }
+}
+
+Map<String, Country> countryTable = {
```

Comment:
Do you think maybe we should move this `countryTable` object into the `CountryTable` class?
So it would be accessed like `CountryTable.table` or `CountryTable.map`, or similarly.
Presently `countryTable` is a global variable without a distinct owner - I think it might help readability a bit in the future if owner of the variable would be obvious.

Reply:
you totally right!